doc/contributing/distribution.md

```diff
@@ -0,0 +1,17 @@
+# Node.js Distribution Policy
+
+This document describes some policies around what is and is not included in the
+Node.js distribution.
+
+## Inclusion
+
+Node.js includes many dependencies, such as V8, Undici and others; and comes
+bundled with standalone tools such as `npm`. It is not a goal of the Node.js
+project to provide a level playing field for all dependencies or bundled tools.
+With limited resources, the Node.js project does not intend to bundle or
+officially support multiple dependencies or tools that serve the same purpose.
+
+## Package managers
```

Comment:
Should we add dependencies to this list?

Reply:
As in, add more sections? Sure, we can, but I figured that this document can always expand over time.

Reply:
Might be worth at least documenting the other ones mentioned earlier in the document - V8 and Undici.